embedded-graphics/CHANGELOG.md

```diff
@@ -36,6 +36,7 @@ Embedded Graphics is a `no_std` library for adding graphics features to display
 - [#429](https://github.com/jamwaffles/embedded-graphics/pull/429) Added `ToBytes` trait to convert colors into byte arrays.
 - [#431](https://github.com/jamwaffles/embedded-graphics/pull/431) Added `MockDisplay::affected_area` to get the area affected by previous drawing operations.
 - [#450](https://github.com/jamwaffles/embedded-graphics/pull/450) Added `ColorConverted` and `DrawTargetExt::color_converted` to support color conversion for draw targets.
+- [#438](https://github.com/jamwaffles/embedded-graphics/pull/438) Added 147 CSS web colors as associated consts to the RGB color types.
```

Comment:
Did we loose some on the way, because there are less than 147 lines in the macro call?

Reply:
We removed 6 `grEy` variants. I've changed this to read "most" instead, but I can put the actual number back in if you think that's better.